lib/scrabble/scrabble-class.rb

```diff
@@ -0,0 +1,61 @@
+module Scrabble
+  class Scrabble
+    SCORES = {
+      1 => ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+      2 => ["d", "g"],
+      3 => ["b", "c", "m", "p"],
+      4 => ["f", "h", "v", "w", "y"],
+      5 => ["k"],
+      8 =>["j", "x"],
+      10 => ["q", "z"]
+    }
+
+    def self.score(word)
+      # Edge case - no input, less than 8 letters
+      if word == "" || word == nil || word.length > 7
```

Comment:
This is fine, but generally you'll want to check nil values first in conditional chains like this. It prevents `NoMethod NilClass errors` that can crop up.